### ISSUE TITLE
accessing "settings" in "pluginRepositories"

### ISSUE THREAD

**Issue by Vampire** [gradle/gradle#1651]
We have a central Ivy repository with all libs and the possibility to replace this with a local copy of the Ivy repository for better offline and home-office-without-VPN workability.

In the `repositories { }` and `buildScript.repositories { }` blocks in build script and settings script this can easily be done, but the following for `pluginRepositories { }` fails because `settings` cannot be accessed:

```
pluginRepositories {
   ivy {
      url settings.hasProperty('publicRepoDir') ? "$publicRepoDir/ivy" : 'http://fileserver.company.com/empic-libs/ivy'
   }
}
```

**Comment by oehme** [comment]
This is fixed in 3.5-rc-1, you can now access the full `Settings` API in the `pluginManagement` block.

**Comment by Vampire** [comment]
Ah, very nice, thanks. :-)
